webapp/packages/plugin-data-viewer/src/TableViewer/TableViewer.tsx

```diff
@@ -99,6 +99,7 @@ interface Props {
   resultIndex: number | undefined;
   presentationId: string | undefined;
   valuePresentationId: string | null | undefined;
+  /** Hide tools panel from Table Viewer */
   simple?: boolean;
```

Comment:
It's not only for the tools panel; it's also possible to disable or hide everything related to the table viewer:
`Display data in simple mode; some features will be hidden or disabled